integration-tests/bats/performance.bats

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+BATS_TEST_TIMEOUT=50
+
```

Comment:
It would be nice to document some context about what the intent of the performance.bats file is. It's obvious to you and me right now, but it likely won't be to future code readers.

Reply:
Added comments.